doc/sections/structure.md

```diff
@@ -13,6 +13,17 @@ Structure of directories and files
    + `vim/`: code for vim
    + `vscode/`: code for vscode
 
+     - `.vscode/*.json`: config for launching, debugging, building the extension
+     - `src/*.ts`: the VSCode plugin source code written in the TypeScript language
+     - `package.json`: the manifest of the plugin (activation events, scripts, dependencies, ...)
+     - `tsconfig.json`: TypeScript configuration (directories, ...)
+     - `vscode.proposed.d.ts`: unstable VSCode API (likely to change)
+     - `lp.configuration.json`: specific characters
+     - `syntaxes/lp.tmLanguage.json`: grammar of lambdapi [WIP : currently only all the reserved keywords are highlighted in one unique color]
```

Comment:
perhaps no need to add "[WIP : ...]"

Reply:
fixed in 59541b70

Reply:
fixed in 609bd5c
(btw, I didn't mention the fact that it will be regularly updated with ''npx vscode-dts dev'' as this file is git tracked and we will do that ourselves)

Reply:
It would be good to add this somewhere, in a editors/vscode/README.md file for instance.